### PR TITLE
feat: Add flex wrapper around repo list content

### LINKED ISSUE #437: Feature: responsiveness in HotRepoCard
### Type of feature

🍕 Feature

### Current behavior

In phone mode it looks very bad, we want to improve our user experience, we are an organization that wants to give the user a good feeling.

### Suggested solution
Screenshot 2023-01-05 at 11 15 36 AM

Create a wrapper
[lines](https://github.com/open-sauced/hot/blob/c44f5c114c3a9860a75d572a87af313085853b7b/src/components/HotRepoCard.tsx#L69&L114)
In small breakpoint you should put ` display flex ` on the wrapper.

### Additional context

_No response_

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Contributing Docs

- [X] I agree to follow this project's Contribution Docs

### PR DESCRIPTION
**Pull request by RozzaysRed**
## What type of PR is this? (check all applicable)

- [x] 🍕 Feature
- [ ] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build
- [ ] 🔁 CI
- [ ] 📦 Chore (Release)
- [ ] ⏩ Revert

## Description

This PR adds responsiveness to the repo list component by adding a flex wrapper around the repo information. 
## Related Tickets & Documents

Fixes #437 

## Mobile & Desktop Screenshots/Recordings

Screenshot 2023-01-18 at 9 41 47 PM
Screenshot 2023-01-18 at 9 44 16 PM

## Added tests?

- [ ] 👍 yes
- [x] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

## Added to documentation?

- [ ] 📜 README.md
- [ ] 📓 docs.opensauced.pizza
- [ ] 🍕 dev.to/opensauced
- [ ] 📕 storybook
- [x] 🙅 no documentation needed

### CODE REVIEW

**Review by TAKANOME-DEV — APPROVED** [review]
Looks great :pizza: Thanks for the contribution @RozzaysRed

File: src/components/RepoList.tsx [hunk 1]
```diff
@@ -42,62 +42,64 @@ const RepoList = ({ data }: RepoListProps): JSX.Element => {
 
   return (
     <div className="flex flex-col gap-y-[20px] md:flex-row bg-white border-[1px] p-[16px] gap-x-[20px] font-Inter border-borderGrey overflow-hidden rounded-[16px]">
-      <div>
-        <div className="rounded-[8px] overflow-hidden w-[88px] h-[88px]">
+      <div className="flex gap-y-[20px] gap-x-[20px] font-Inter overflow-hidden w-full">
```
Comment (NsdHSO) [inline src/components/RepoList.tsx:45]: Can you instead `gap-y-[20px]`  use `gap-y-5` and instead `gap-x-[20px]` use `gap-x-5'

**Review by NsdHSO — COMMENTED** [review]
LGTM

File: src/components/RepoList.tsx (hunk 1 above)
Comment (RozzaysRed) [reply src/components/RepoList.tsx]: Sure thing, I also adjusted similar classes in the parent div as well.

**Comment by NsdHSO** [comment]
LGTM

File: src/components/RepoList.tsx [hunk 2]
```diff
@@ -85,20 +101,38 @@ const RepoList = ({ data }: RepoListProps): JSX.Element => {
...
-              className="w-[16px]"
-            />
 
-            <p className="text-sm">
-              {humanizeNumber(stars)}
+            <p className="text-base text-textGrey">
+              {description}
             </p>
           </div>
+          </a>
```
Comment (bdougie) [inline src/components/RepoList.tsx:109]:
```suggestion
```

**Comment by bdougie** [comment]
My attempt at fix merge conflicts has broken this. @RozzaysRed let me know if you have bandwidth to take a look at this agin.

**Comment by RozzaysRed** [comment]
@bdougie sure, I can work on getting figuring out what's going on.

**Comment by RozzaysRed** [comment]
@bdougie should be addressed now. 
Screenshot 2023-05-20 at 7 39 01 PM

**Review by bdougie — APPROVED** [review]
Thanks

**Pull request merged**